measure_extinction/extdata.py

```diff
@@ -1056,38 +1062,105 @@ def plot(
                     fontsize=fontsize,
                 )
 
+    def fit_band_ext(self):
+        """
+        Fit the observed NIR extinction curve with a powerlaw model, based on the band data between 1 and 40 micron
+
+        Parameters
+        ----------
+
+        Returns
+        -------
+        Updates self.model["waves", "exts", "params"] and self.columns["AV"] with the fitting results:
+            - waves: np.ndarray with the wavelengths used in the fitting
+            - exts: np.ndarray with the fitted powerlaw model to the extinction curve
+            - params: tuple with the parameters (amplitude, alpha) if data in A(lambda)/A(V) or (amplitude, alpha, A(V)) if data in E(lambda-V)
+        """
+        # retrieve the band data to be fitted
+        ftype = "BAND"
+        gbool = np.all(
+            [
+                (self.npts[ftype] > 0),
+                (self.waves[ftype] > 1.0 * u.micron),
+                (self.waves[ftype] < 40.0 * u.micron),
+            ],
+            axis=0,
```

Comment:
This is very specific to SpeX.  *After* you have your talk done, I would suggest that this code is a good candidate for moving to a separate repository specifically for the SpeX project.

Reply:
Really this whole function given it also has a model that is specific to the NIR fitting.

Reply:
Ok. I opened an issue for this: #68